static/app/views/issueList/searchBar.tsx

```diff
@@ -64,6 +68,11 @@ const getFilterKeySections = (
     allTags.filter(tag => tag.kind === FieldKind.EVENT_FIELD),
     ['key']
   ).map(tag => tag.key);
+  const eventTags = orderBy(
+    allTags.filter(tag => tag.kind === FieldKind.TAG),
+    ['totalValues', 'key'],
+    ['desc', 'asc']
+  ).map(tag => tag.key);
```

Comment:
Tags retrieved from the organization/org-slug/tags endpoint (which have `kind=FieldKind.TAG`) fall into the Event Tags section in the search dropdown